documentation/directives.md

```diff
@@ -3,6 +3,14 @@ title: "SDL directives"
 date: 2018-09-09T12:52:46+10:00
```

Comment:
As with all our documentation there are v22 and master versions, however these happen to be the same information

I renamed this filed from `sdl-directives` to `directives` to make it clearer it contains both schema and query/operation directives

Reply:
The diff has gotten confused about mapping which files got renamed, don't worry they are identical